go-app-ussd_popi_user_data.js

```diff
@@ -155,33 +138,7 @@ go.app = function() {
                 }
         };
 
-        // override normal state adding
-        self.add = function(name, creator) {
-            self.states.add(name, function(name, opts) {
-                if (!interrupt || !utils.timed_out(self.im))
-                    return creator(name, opts);
-
-                interrupt = false;
-                var timeout_opts = opts || {};
-                timeout_opts.name = name;
-                return self.states.create('state_timed_out', timeout_opts);
-            });
-        };
-
-        self.states.add('state_timed_out', function(name, creator_opts) {
-            return new ChoiceState(name, {
-                question: $('Welcome back. Please select an option:'),
-                choices: [
-                    new Choice('state_start', $('Main menu'))
-                ],
-                next: function(choice) {
-                    return choice.value;
-                }
-            });
-        });
```

Comment:
Same question here with the 182/140 character limit and whether we've checked text + translations.